torch/nn/modules/loss.py

```diff
@@ -1286,6 +1287,24 @@ class CTCLoss(_Loss):
         >>> loss = ctc_loss(input, target, input_lengths, target_lengths)
         >>> loss.backward()
 
+    Example2::
```

Comment:
See above.

Reply:
its my honor to get this suggestions from you